docs/running-on-mesos.md

```diff
@@ -516,6 +516,15 @@ See the [configuration page](configuration.html) for information on Spark config
     Fetcher Cache</a>
   </td>
 </tr>
+<tr>
+  <td><code>spark.mesos.unavailabilityThreshold</code></td>
```

Comment:
Yes, unavailability threshold is supposed to be the out best estimate for how long a task takes to run. 
ex: 
let `o1` be an offer with unavailability start at `10`. 
the scheduler receives `o1` at `5`, 

If the threshold was set to 5 (our estimate of how long a task might take), then a task is scheduled to use that offer, as we know that the task will end before the start of maintenance.
Whereas if the threshold was 6, that offer is rejected as it could still be running the task when   maintenance is started